easybuild/easyconfigs/c/Cartopy/Cartopy-0.19.0.post1-foss-2020b.eb

```diff
@@ -0,0 +1,51 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Updated: Denis Kristak
+easyblock = 'PythonBundle'
+
+name = 'Cartopy'
+version = '0.19.0.post1'
+
+homepage = 'https://scitools.org.uk/cartopy/docs/latest/'
+description = """Cartopy is a Python package designed to make drawing maps for data analysis and visualisation easy."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Fiona', '1.8.20'),
+    ('GDAL', '3.2.1'),
+    ('GEOS', '3.9.1'),
+    ('matplotlib', '3.3.3'),
+    ('pyproj', '3.0.1'),
+    ('SciPy-bundle', '2020.11'),
+    ('Shapely', '1.8a1'),
+    ('lxml', '4.6.2'),
+    ('Pillow', '8.0.1'),
+    ('PROJ', '7.2.1'),
+    ('PyYAML', '5.3.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = True
```

Comment:
`download_dep_fail` is enabled by default for `PythonBundle`, so please remove (required to make easyconfigs test suite pass)

Reply:
yep